README.md

```diff
@@ -269,6 +270,30 @@ You can run all the unit tests using nosetests. Most of the tests are mocked.
 $ nosetests --verbosity=2 tests/
 ```
 
+
+## Local Development
+
+There are many ways to authenticate with Cerberus and they don't always work on local. Luckily you can easily get a token from the dashboard and export it to your environment variables. Doing so will override the authentication mechanism that's set in place. For example:
+```python
+from cerberus.client import CerberusClient
+client = CerberusClient('https://dev.cerberus.nikecloud.com') # This will fail on local when it tries to call the metadata endpoint.
+```
+Without changing any code, you can do:
+```bash
+$ export CERBERUS_TOKEN='mytoken'
+```
+```python
+from cerberus.client import CerberusClient
+client = CerberusClient('https://dev.cerberus.nikecloud.com') # On local, the client will pick up the environment variable that was set earlier. When it's deployed to an EC2 instance, it'll automatically switch to authenticating using the metadata endpoint.
```

Comment:
This is not necessarily true right?

If the client is deployed on an EC2 instance, but the token environment variable is set, then the environment variable will still be used

Reply:
addressed